README.md

```diff
@@ -47,6 +47,18 @@ WebNN requires a compatible browser to run, and Windows 11 v21H2 (DML 1.6.0) or
 4. In the drop-down menu, select `Enabled`. 
 5. Relaunch your browser.
 
+#### Running WebNN on NPU
+Running WebNN on NPU requires capable NPU hardware e.g. Intel® AI Boost NPU from Intel® Core™ Ultra processors.
+
+1. Download and install the [latest Intel NPU driver](https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html).
+2. Microsoft Edge browser will download the latest DirectML.dll automatically when WebNN is enabled. Apply the latest redistributable DirectML.dll for Google Chrome Canary:
```

Comment:
It seems this step and the following command line step are unnecessary for Edge. I would recommend to separate steps for Chrome and Edge, make the DirectML.dll downloading, deploying and command line steps just for Chrome.

Reply:
Updated, PTAL, @ibelem @huningxin